src/winforms/toga_winforms/factory.py

```diff
@@ -17,9 +17,10 @@
 from .widgets.scrollcontainer import *
 from .widgets.selection import Selection
 from .widgets.slider import *
-from .widgets.splitcontainer import *
-from .widgets.switch import *
+from .widgets.splitcontainer import SplitContainer
+from .widgets.switch import Switch
 from .widgets.table import *
```

Comment:
At column 1: (F403) 'from .widgets.table import *' used; unable to detect undefined names